core/formats/sparse_bitmap.cpp

```diff
@@ -403,10 +512,12 @@ void sparse_bitmap_iterator::read_block_header() {
     cont_begin_ = in_->file_pointer() + block_size;
     ctx_.u8data = in_->read_buffer(block_size, BufferHint::NORMAL);
 
-    // FIXME check alignment
-    seek_func_ = ctx_.u8data
-      ? &container_iterator<type>::seek<AT_DIRECT>
-      : &container_iterator<type>::seek<AT_STREAM>;
+    ResolveType(
+        [this]<AccessType A, bool TrackPrev>() noexcept {
+          seek_func_ =
+              &container_iterator<BT_DENSE, TrackPrev>::template seek<A>;
+        },
+        ctx_.u8data, track_prev_doc_);
```

Comment:
make for sparse_bitmap_iterator function: set_seek_func
```suggestion
    set_seek_func<BT_DENSE>();
```

Reply:
Done